pkg/clients/cloud_client.go

```diff
@@ -115,6 +116,39 @@ func (c *CloudAPIClient) GetRegionDetails(ctx context.Context, provider CloudPro
 	return &region, nil
 }
 
+// EnableRegion sends a PATCH request to enable a region and polls until the region is enabled or a timeout is reached.
```

Comment:
It seems like this comment describes the region resource, this function doesn't appear to do any polling?

Reply:
Ah yes, I initially had it setup that way, but a better approach is to do the polling in the resource itself rather than the client lib, so I moved the `waitForRegionToBeEnabled` to the resource itself. Will update that comment accordingly!